style.css

```diff
@@ -0,0 +1,159 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Open Sans", sans-serif;
+}
+/* 
+  style the Header section 
+  */
+header {
+  background-color: #f8f8f8;
+  border-color: #e7e7e7;
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  height: 40px;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  border-style: solid;
+  border-width: thin;
+  padding: 0px 15px;
+}
+nav {
+  width: 100%;
+}
+
+/* Style the links inside the header section */
+ul li a {
+  text-decoration: none;
+  cursor: pointer;
+  color: #777777;
+}
+
+.logo {
+  width: 265px;
+  height: 50px;
+  cursor: pointer;
+}
+
+/*  style the elemints inside the navegation bar */
+#benefit-item,
+#blog-item,
+#login-item,
+#resister-item,
+#work-item {
+  height: 40px;
+  font-size: 13px;
+  border-style: solid;
+  border-width: thin;
+  border-color: #d3d3d3;
+  border-top: none;
+  text-align: center;
+  font-weight: 400px;
+  display: block;
+}
+.items {
+  /* margin-left: 60px; */
+  list-style: none;
+  display: flex;
+}
+#benefit-item {
+  width: 80.46px;
+}
+#blog-item {
+  width: 63.89px;
+}
+#work-item {
+  width: 125.13px;
+}
+#login-item {
+  width: 64px;
+}
+#resister-item {
+  width: 80.22px;
+}
+
+.landing-container {
+  position: relative;
+}
+.landing-img {
+  width: 100%;
+  height: 400px;
+  margin-top: 39px;
+}
+.menu-icon {
+  display: none;
+}
+
+/* style the content inside the pictures */
+.content {
+  position: absolute;
+  text-align: center;
+  top: 50%;
+  left: 50%;
+
+  transform: translate(-50%, -50%);
+}
+h2 {
```

Comment:
same here, don't give the style to the element directly, suppose one of your teammates used h2 in their code it will take your style. so it would be better to give the element a class name or id then style it using them.